src/OnboardingSPA/components/Drawer/stylesheet.scss

```diff
@@ -422,7 +415,7 @@ $main-border-main--rgb: var(--nfd-onboarding-highlighted--rgb);
 		}
 
 		&__text {
-			color: $main-color-dark;
+			color: var(--nfd-onboarding-black);
```

Comment:
So currently there is no variable per brand for the color black ... it is not need since if its black across brands it can be defined in a common location ... that is why its defined in the _interface.css and should be used that way only